libmscore/repeatlist.cpp

```diff
@@ -30,7 +30,7 @@ Volta* Score::searchVolta(int tick) const
       {
       for (const std::pair<int,Spanner*>& p : _spanner.map()) {
             Spanner* s = p.second;
-            if (s->type() != ElementType::VOLTA)
+            if (!s->isVolta())
```

Comment:
I believe this and some more `isXxx()` and `toXxx()` won't fit for 2.2.

Reply:
indeed.. if desired by @lasconic I could rework this into two separate commits, then the first one would remain 2.x compatible and all "codestyle" changes into a 2nd commit?

Reply:
I can handle this. Give me a couple of days to try the PR and see how much it improve  the current situation and break compatibility.